stdlib/_ast.pyi

```diff
@@ -67,8 +87,45 @@ class FunctionDef(stmt):
     body: list[stmt]
     decorator_list: list[expr]
     returns: expr | None
+    type_comment: str | None
     if sys.version_info >= (3, 12):
         type_params: list[type_param]
+        @overload
+        def __init__(
+            self,
+            name: _Identifier,
+            args: arguments,
+            body: list[stmt],
+            decorator_list: list[expr],
+            returns: expr | None,
+            type_comment: str | None,
+            type_params: list[type_param],
+            **kwargs: Unpack[_Attributes],
+        ) -> None: ...
+        @overload
+        def __init__(
+            self,
+            name: _Identifier,
+            args: arguments,
+            body: list[stmt],
+            decorator_list: list[expr],
+            returns: expr | None = None,
+            *,
+            type_comment: str | None = None,
```

Comment:
I don't think type_comment needs to be keyword-only on this overload.

Reply:
Addressed in https://github.com/python/typeshed/pull/11880/commits/9655775384bf9ae8b2c69e4b07f0baafd0dbaf3e